docs/.template.md

```diff
@@ -27,6 +28,9 @@ The [Control Name](API-Link) ...
 > Some warning note
 -->
 
+> [!div class="nextstepaction"]

```

Comment:
I'm just curious, where is this style defined?  I'm assuming it's just part of docs.microsoft.com, but I didn't seem to be able to find it.

Reply:
It's a custom extension on docs.microsoft.com.
Having seen similar things in other sites, I think it's licensed from a 3rd party CMS system and so couldn't be included in public repos.
X-Ref https://docs.microsoft.com/en-us/contribute/markdown-reference#next-step-action